x/ecocredit/module/module.go

```diff
@@ -35,23 +35,23 @@ type Module struct {
 }
 
 // NewModule returns a new Module object.
-func NewModule(paramSpace paramtypes.Subspace, accountKeeper ecocredit.AccountKeeper, bankKeeper ecocredit.BankKeeper) Module {
+func NewModule(paramSpace paramtypes.Subspace, accountKeeper ecocredit.AccountKeeper, bankKeeper ecocredit.BankKeeper) *Module {
```

Comment:
Pointer receiver is required to handle keeper registration in `RegisterServices`